README.md

```diff
@@ -130,50 +129,46 @@ on-demand as we go along.
 ### Pipelining
 
 Expression provides a `pipe` function similar to `|>` in F#. We don't want to
-overload any Python operators e.g `|` so `pipe` is a plain old function taking
+overload any Python operators, e.g., `|` so `pipe` is a plain old function taking
 N-arguments, and will let you pipe a value through any number of functions.
 
-```py
+```python
 from expression import pipe
 
-gn = lambda g: g * y
```

Comment:
This example was confusing. `x` wasn't defined, y and z weren't defined but were used in the lambdas.